src/thunderbots/software/ai/world/team.cpp

```diff
@@ -1,33 +1,145 @@
 #include "team.h"
+#include <set>
+#include "shared/constants.h"
```

Comment:
nit; but could we keep our projects headers together and seperate from the std headers?

Reply:
I thought clang-format would have taken care of this, but it seems to only sort alphabetically and not also by "type" of header. Will look into possible solutions, preferably automated.